### PR TITLE
See output in Cucumber

### PR DESCRIPTION
**Pull request by kevgo**

### CODE REVIEW

File: test/steps/suite_steps.go [hunk 1]
```diff
@@ -65,3 +74,35 @@ func scenarioName(args interface{}) string {
...
+func scenarioTags(args interface{}) []*gherkin.Tag {
+	scenario, ok := args.(*gherkin.Scenario)
+	if ok {
+		return scenario.Tags
+	}
+	scenarioOutline, ok := args.(*gherkin.ScenarioOutline)
+	if ok {
+		return scenarioOutline.Tags
+	}
+	panic("unknown scenario type")
```
Comment (charlierudolph) [inline test/steps/suite_steps.go]: Can we use a type switch here: https://tour.golang.org/methods/16

File: test/shell_runner.go [hunk 2]
```diff
@@ -90,7 +91,13 @@ func (runner *ShellRunner) Run(name string, arguments ...string) (output string,
 	cmd.Dir = runner.dir
 	cmd.Env = customEnv
 	rawOutput, err := cmd.CombinedOutput()
-	return string(rawOutput), err
+	output = string(rawOutput)
+	if Debug {
+		fmt.Printf("In %q:\n", path.Base(runner.dir))
```
Comment (charlierudolph) [inline test/shell_runner.go]: Thoughts on using the format we used in ruby where the dir shares the line with the command?
```
dir > name args...
```
https://github.com/Originate/git-town/blob/master/features/support/run_helpers.rb#L48

Comment (kevgo) [reply test/shell_runner.go]: Good idea, done.

File: test/steps/suite_steps.go (hunk 1 above)
Comment (kevgo) [reply test/steps/suite_steps.go]: That's a pretty cool trick! Definitely makes the code look cleaner.

**Comment by kevgo** [comment]
All comments addressed.

**Pull request merged**